index.html

```diff
@@ -0,0 +1,146 @@
+            <div class="headline">
+                <h1 id="headlineSAFOR">SAFOR</h1>
+                <h1>switching your<br>
+                    life to safety mode</h1>
+                <p>Safor is an intelligent personal guard platform that empowers individuals to enjoy active and safe
+                    lives,<br>
+                    while their friends and family can enjoy peace of mind.</p>
+            </div>
+            <div id="hrs">
+                <hr id="hr1">
+                <hr id="hr2">
+                <hr id="hr3">
+                <hr id="hr4">
+                <hr id="hr5">
+                <hr id="hr6">
+                <hr id="hr7">
+            </div>
+        </div>
+    </div>
+    <div class="two">
+        <div id="about">
+            <div>
+                <h1>WHAT IS SAFOR?</h1>
+                <div>
+                    <img src="assets/Rectangle.svg">
+                    <h3 id="abouth3"><a href=" ">WATCH NOW</a></h3>
+                </div>
+            </div>
+        </div>
+        <div class="aboutsafor">
+            <p> The existing types of protection in today's society can not always
+                be adapted to the needs of the modern customers.
+                <br><br>
+                The SAFOR system is flexible and adaptive, providing personal protection to you and your loved once at
+                any time of the day, wherever they are.</p>
+            <p id="aboutp">Regardless the situation, the system
+                has unlimited possibilities to make your life free and safe.</p>
+        </div>
+    </div>
+    <div class="three" id="howitworks">
+        <div class="threetext">
+            <h3>HOW IT WORKS</h3>
+            <h2 id="threeh2border">Install app IOS / ANDROID</h2>
+            <img src="assets/border.svg">
+            <h2 id="threeh2">Complete<br>
+                the registration</h2>
+            <h2>SECURE<br>
+                YOUR LIFE IN ONE TAP</h2>
+            <p id="threeshortp">Additional</p>
+            <p id="threelongp">Set up your home and office locations to simplify your search in the city.
+                Add your family contact to notify them in dangerius situation</p>
+        </div>
+        <div>
+            <img src="assets/Bitmap.png" id="threeimg1">
+            <img src="assets/Bitmap2.png" id="threeimg2">
+        </div>
+        <div class="four">
+            <h3>WHY SAFOR?</h3>
+            <h2>Reliability</h2>
+            <p>You do not need to worry that you will be far from the city or you will be hard to find in a
+                urbanjungle
+            </p>
+            <h2>Speed</h2>
+            <p>Do not waste time explaining the situation and describing what is happening</p>
+            <h2>Anonymity</h2>
+            <p>Your relatives can be notified that you need help only with your consent</p>
+        </div>
+    </div>
+    <div class="five">
+        <div class="forbusiness" id="forbusiness">
```

Comment:
ID here is useless, please use only a class. Also please review the document, a lot of parts doesn't seem to be needed (especially a combo with id & class with the same name).